Languages/de.ini

```diff
@@ -535,10 +535,6 @@ Clear Credits=♥  Credits Löschen
 EditorNoteSkin=Editor Notenskin
 
 
-# GraphicsSoundOptions
-VideoRenderer=Video Renderer

```

Comment:
I haven't checked, but why are we removing this string just for de.ini and not others? Do we need to remove it?

Reply:
Ah, I have a reasoning for that. Let me explain:

The `VideoRenderer` key in `de` contained the english string, so I thought it's better to remove it instead of taking the risk that it will diverge from the actual `en` string in the future.

The `de` translation has a lot of `en` keys in there. There already is an automatic fallback to `en` in the code, so duplicating untranslated keys is not necessary. `de` is as far as I can tell the only translation that duplicates untranslated strings on that large of a scale (on a quick glance I found a couple of en string in the jp translation, but not much). I didn't want to make the commit about cleaning out the untranslated strings from `de`, so I only did that for the part of the code that I touched.

All the other translations either have a proper translation for `VideoRenderer` or don't contain the key at all.

Does that make sense to you?

Reply:
Ah yes that makes sense!